plugins/out_stackdriver/stackdriver.c

```diff
@@ -696,6 +696,38 @@ static int get_severity_level(severity_t * s, const msgpack_object * o,
     return -1;
 }
 
+static int get_stream(msgpack_object_map map)
+{
+    int i;
+    int len_stream;
+    int val_size;
+    msgpack_object k;
+    msgpack_object v;
+
+    len_stream = sizeof(STDOUT) - 1;
```

Comment:
why do we have this calculation?

Reply:
it's a fixed value used later to compare if a key value matches, comparing length first for longer maps is faster than doing strncmp(), since strncmp implies string comparison, most of the cases the conditional can be improved by comparing length first

Reply:
Thanks for the explanation! 

I meant why do we use the length of `stdout` str instead of `stream` str...

Reply:
Oops I think the naming makes some confusion here. I will rename it to be len_stdout.